x-pack/legacy/plugins/fleet/scripts/dev_agent.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import yargs from 'yargs';
```

Comment:
Are you open to using our `run()` helper from `@kbn/dev-utils`? https://github.com/elastic/kibana/tree/master/packages/kbn-dev-utils/src/run

We're trying to standardize the little CLIs that are scattered about.

Reply:
No problem I am going to make the change 👍